WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -11009,7 +11009,6 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 1;
```

Comment:
By removing this and the `MARKETING_VERSION` overrides (three times for Debug, Release, Release-Alpha), we let the target inherit the setting from the top-most `xcconfig`:

![image](https://user-images.githubusercontent.com/1218433/188559372-27beca9e-e8ab-46dc-ba37-9da7a49f65ca.png)